src/Console/AbstractCommand.php

```diff
@@ -114,28 +114,32 @@ protected function initDbConnection()
     /**
      * Correctly write output messages when a progress bar is displayed.
      *
-     * @param string|array $messages
-     * @param ProgressBar  $progress_bar
-     * @param integer      $verbosity
+     * @param string|array      $messages
+     * @param ProgressBar|null  $progress_bar
+     * @param integer           $verbosity
      *
      * @return void
      */
     protected function writelnOutputWithProgressBar(
         $messages,
-        ProgressBar $progress_bar,
+        ?ProgressBar $progress_bar,
```

Comment:
metod name is `WithProgressBar`, but porgress bar is optional? seems a nonsense

Reply:
also, seems out of the scope of current PR, isn't it?

Reply:
Changed in latest commit.